Kwartracker/Resources/fil-PH.lproj/Localizable.strings

```diff
@@ -71,3 +71,7 @@
 "login_and_security_page.label.email" = "Email Address";
 "login_and_security_page.dummy_data.email" = "email_email_email@emailemail.com";
 "login_and_security_page.dummy_data.password" = "***************";
+
+// Categories pages
+"category_page.navigation_button_item.add_icon" = "AddIcon";
```

Comment:
No white space or space?? I think it should be `Add Icon`??

Reply:
animationSpeed